_docs/writing-style-guidelines.md

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
We shouldn't list all of this twice. It should either be in the main readme or here, but not both.

Reply:
Are you proposing we shouldn't keep this file in the main website content?  My concern with having it only in the README is that some contributors might not read the readme file...

Reply:
I would not expect this content to change often...